### PR TITLE
Rewrite temporary message inheritance

### PR DESCRIPTION
**Pull request by carolinegreen**
Remove temporary message section, and assign this to the temporary-message class which is clearer.  Also add inheritance from this variable to temporary-message-main class so that class has correct margin.

### CODE REVIEW

File: toolkit/scss/_temporary-message.scss [hunk 1]
```diff
@@ -7,18 +7,14 @@
...
 
-%temporary-message-section {
-  @extend %temporary-message;
-  margin-bottom: $gutter;
-}
-
+%temporary-message-section,
 .temporary-message {
-  @extend %temporary-message-section;
+  @extend %temporary-message;
```
Comment (pcraig3) [inline toolkit/scss/_temporary-message.scss]: Do you think it's worth importing this in this file?  

As it stands, it seems like a hidden dependency on the `shared_placeholders/_temporary-messages.scss` file.

Comment (carolinegreen) [inline toolkit/scss/_temporary-message.scss]: Yes, it this import wasn't something I added, and other files are using this.

Comment (pcraig3) [inline toolkit/scss/_temporary-message.scss]: I mean, do you think it's worth adding an `@import "shared_placeholders/_temporary-messages.scss";` statement at the top of this file?

Comment (carolinegreen) [inline toolkit/scss/_temporary-message.scss]: I think the problem is that extend is being used and if the shared_placeholders.scss file is imported again further down by another file, the changes in the extended class will be overwritten.  This has already surfaced as a problem in the buyer app, where the shared sass placeholder partial was introduced after the temporary message partial.

I think the best thing is probably to leave it just now, but I will put a story in pivotal tracker for looking at how imports are being handled in general.  Even imports are left as they are, it probably is still worth putting in documentation to explain the decisions behind how they are being handled.

**Comment by pcraig3** [comment]
👍

File: toolkit/scss/_temporary-message.scss (hunk 1 above)
Comment (pcraig3) [inline toolkit/scss/_temporary-message.scss]: Ah, makes sense.  Sounds a bit convoluted, but w/e.

Comment (carolinegreen) [inline toolkit/scss/_temporary-message.scss]: Imports still aren't great in sass sadly.

Comment (carolinegreen) [inline toolkit/scss/_temporary-message.scss]: The way sass handles imports still isn't great unfortunately.

**Pull request merged**